edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/edgedeployment/PodExtensions.cs

```diff
@@ -5,7 +5,8 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.EdgeDeployment
     using System.Linq;
     using k8s.Models;
     using Microsoft.Azure.Devices.Edge.Agent.Core;
-    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Org.BouncyCastle.Security;

```

Comment:
Seems suspicious...

Reply:
I think VS added this, another import got added in my new iteration without me doing anything (that I known of).  

---
In reply to: [341775711](https://github.com/Azure/iotedge/pull/1938#discussion_r341775711) [](ancestors = 341775711)

Reply:
omg it came back wtf, I removed this locally!

---
In reply to: [341783173](https://github.com/Azure/iotedge/pull/1938#discussion_r341783173) [](ancestors = 341783173,341775711)